aptos-move/block-executor/src/executor.rs

```diff
@@ -1114,14 +1256,31 @@ where
                             &latest_view,
                         );
 
+                        // TODO[agg_v2]: patch group writes in finalized groups
+                        let combined_groups = Self::combine_groups(finalized_groups)
+                            .map_err(|e| Error::FallbackToSequential(e))?;
```

Comment:
Why fallback? Because we disable delayed fields there? We are in sequential already

Reply:
yeah, the same debacle as above.